scripts/linux/install-chrome.sh

```diff
@@ -14,7 +14,10 @@ if [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
 
   # Installs chromedriver for Linux 64 bit systems.
   LATEST_CHROMEDRIVER=$(wget -q -O - http://chromedriver.storage.googleapis.com/LATEST_RELEASE)
-  wget -N http://chromedriver.storage.googleapis.com/$LATEST_CHROMEDRIVER/chromedriver_linux64.zip
+  # wget -N http://chromedriver.storage.googleapis.com/$LATEST_CHROMEDRIVER/chromedriver_linux64.zip
+  # Temporarily pin to Chromedriver 74.
+  # @see https://github.com/acquia/blt/issues/3704
+  wget -N https://chromedriver.storage.googleapis.com/74.0.3729.6/chromedriver_linux64.zip
```

Comment:
Just a thought here: it might make sense to make the version value an environment variable with `74.0.3729.6` as its default for now. This would make it easier for projects using this script to set a different version.